### PR TITLE
openshift-cli 4.11.0

### PR DESCRIPTION
**Pull request by georgettica**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by georgettica** [comment]
#98541

File: Formula/openshift-cli.rb
Comment (carlocab) [inline Formula/openshift-cli.rb:9]: It looks to me like the date part of the tag should be part of the version. Otherwise, we won't be able to ship newer tags that differ only in the date.

Comment (georgettica) [reply Formula/openshift-cli.rb:9]: worth a try on my end, edited manually :)

**Comment by georgettica** [comment]
I think this error is a bit suprising. not sure how to tackle it 😱

**Comment by Bo98** [comment]
Remove the patch since I think it's integrated into this release now.

**Comment by georgettica** [comment]
Removed patch, let's see how it goes :)

File: Formula/openshift-cli.rb
Comment (georgettica) [reply Formula/openshift-cli.rb:9]: I'll close it when I have a passing build, as so people coming in will be able to see the trial and error done :)

**Comment by georgettica** [comment]
I am not sure how this became the case (with livecheck) but LMK on what to do now :/

File: Formula/openshift-cli.rb
Comment (Bo98) [inline Formula/openshift-cli.rb:11]: Looks like we're able to auto-parse this from the tag so this line can be removed.

**Comment by Bo98** [comment]
> I am not sure how this became the case (with livecheck) but LMK on what to do now :/

The regex just needs adjusting to include the date.

Maybe try `/^openshift-clients[._-](\d+(?:\.\d+)+(?:[._-]p?\d+)?)$/i`

File: Formula/openshift-cli.rb
Comment (georgettica) [reply Formula/openshift-cli.rb:11]: So remove the tag?
No problemo!

**Comment by georgettica** [comment]
@Bo98 did a copy-pasta and it seems like this was not changed much.
Let me know if I can replace the regex further

**Comment by georgettica** [comment]
Let me squash all commits so it's ready to merge :)

**Comment by georgettica** [comment]
Also, it seems the build failure tag doesn't go away

**Comment by georgettica** [comment]
Thanks all!
It compiles and tests are passing now.
Just need to get approval and it can be closed

**Review by carlocab — APPROVED** [review]
Thanks!

**Comment by georgettica** [comment]
@carlocab 
for some reason I am getting this error:
```
$ brew install openshift-cli
==> Downloading https://ghcr.io/v2/homebrew/core/openshift-cli/manifests/4.10.0-2
curl: (22) The requested URL returned error: 404

Error: openshift-cli: Failed to download resource "openshift-cli_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/openshift-cli/manifests/4.10.0-2
```

**Comment by georgettica** [comment]
just a maintainer issue:
```
$ brew doctor
...
Warning: Some taps are not on the default git origin branch and may not receive
updates. If this is a surprise to you, check out the default branch with:
  git -C $(brew --repo homebrew/core) checkout master
```

**Comment by georgettica** [comment]
not sure though.. should the version be so low?
```
$ oc version
Client Version: v4.2.0-alpha.0-1420-gf1f09a3
```

**Comment by carlocab** [comment]
> not sure though.. should the version be so low?
> 
> ```
> $ oc version
> Client Version: v4.2.0-alpha.0-1420-gf1f09a3
> ```

That's a known issue: #76968

**Pull request closed without merging**
